addons/web_editor/static/src/js/editor/odoo-editor/src/utils/utils.js

```diff
@@ -2766,22 +2814,3 @@ export function cleanZWS(node) {
         .filter(node => node.nodeType === Node.TEXT_NODE)
         .forEach(node => node.nodeValue = node.nodeValue.replace(/\u200B/g, ''));
 }
-
-export function getOffsetAndCharSize(nodeValue, offset, direction) {
```

Comment:
Why do we remove this utility function ?
If it's not used anymore we could indeed remove it, but I don't see any other reference to it in this commit. So it should probably have it's own commit if the removal is legit.

Reply:
We can't remove it in stable in case of user customizations that might rely on it.

Reply:
My bad, 
I was looking at the commit one by one and probably confused the diff Github gave me. Don't worry about it.